wa/framework/uiauto-androidx/app/src/main/java/com/arm/wa/uiauto/ActionLogger.java

```diff
@@ -0,0 +1,60 @@
+/*    Copyright 2014-2018 ARM Limited
```

Comment:
I left it the same because the file is an identical copy in this case, but I can update if you think it would be more appropriate.

Reply:
If you don't mind updating the files you're adding/touching it would be appreciated. Technically we should be updating the notice for each file we update however we've not been keeping this up to date.

Reply:
Let me get those updated then.